distribution/docs/src/main/resources/content/_appReferences/mg-geowebcache.adoc

```diff
@@ -48,42 +48,37 @@ Add layers to the local cache:
 . If necessary, click the *Add* button to add additional WMS Layer Names.
 
 .Add Layer
-[cols="1,1m,1,2,1,1" options="header"]
+[cols="1,1m,1,2,1" options="header"]
 |===
 |Name
 |Property
 |Type
 |Description
 |Default Value
-|Required
```

Comment:
❓ What is the motivation for taking out the `Required` table entry?

Reply:
I don't think that it provides any useful information for the person doing the configuring.